src/promptflow/tests/sdk_cli_test/unittests/test_exceptions.py

```diff
@@ -172,13 +179,13 @@ def test_error_category_with_cause_exception3(self, pf):
                 raise Exception("pf run failed") from e
         except Exception as e:
             ex = e
-        error_category, error_type, error_target, error_message = _ErrorInfo.get_error_info(ex)
-        assert error_category == ErrorCategory.UserError
-        assert error_type == "FileNotFoundError"
+        error_category, error_type, error_target, error_message, error_detail = _ErrorInfo.get_error_info(ex)
+        assert error_category == ErrorCategory.UNKNOWN
+        assert error_type == "Exception"
         assert error_target == ErrorTarget.UNKNOWN
+        assert error_message == ""
         assert (
-            "exception msg=, "
-            "exception module=promptflow._sdk._pf_client, "
-            'exception code=raise FileNotFoundError(f"flow path {flow} does not exist"), '
-            "exception lineno="
-        ) in error_message
+            "exception module=Non promptflow module, not recorded., "
```

Comment:
I think we can omit the exception prefix?

Reply:
ok, done.